### ISSUE TITLE
Issue with page: What Is The edX Accessibility Best Practices Guidance Based On?

### ISSUE THREAD

**Issue by srushe1** [openedx/docs.openedx.org#648]
### Filename

concepts/accessibility/accessibility_guidelines.rst

### What persona(s) does this page touch? (skip if you don't know)

Educators, Documentors

### What is broken about this file?

Other (detail below)

### Provide any extra details, such as LMS version, specifics of what's broken, or suggested remediation steps.

There is a note in this file that says: This content has been migrated from legacy edx.org docs and has not yet been adjusted for the Open edX® Community. 

Should be reviewed and adjusted by the appropriate people (note: other files related to accessibility have notations saying it needs to be reviewed by legal, I would anticipate that this page will require the same legal review).
